### ISSUE TITLE
FixedOffsetTimezone repr() is awkard for negative offsets

### ISSUE THREAD

**Issue by moreati** [mitsuhiko/babel#215]
```python
>>> babel.util.FixedOffsetTimezone(-60)
<FixedOffset "Etc/GMT+-60" -1 day, 23:00:00>
```

Ideally this would be something like `<FixedOffset "Etc/GMT-60" -1:00:00>`. I'm not sure it matters enough to change the existing behaviour. Assessments and recommendations welcomed.

**Comment by sils1297** [comment]
Indeed, I'm for this change but it's not a real bug so it doesn't need backporting.

**Comment by akx** [comment]
The +-60 bug was fixed by d816803400444d116ec285c38d6367d08a17b374, it seems.

I'm wondering whether FixedOffsetTimezone should exist in Babel at all...
